p2p/nodeconfig/config.go

```diff
@@ -85,3 +85,39 @@ type TimeConfig struct {
 	DefaultTimeoutLatency duration
 	RefreshNtpInterval    duration
 }
+
```

Comment:
Again here we can remove `TimeConfigValues` and `SwarmConfigValues` from above now that we have `DefaultConfig`.